pkgs/test_api/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.11
```

Comment:
If we are publishing this we will also need to publish the other ones, might as well prep them in the same CL.